### PR TITLE
Misc changes to accommodate Ben's changes so bundle can be used as is. 

### PR DESCRIPTION
**Pull request by praveenkumar**
- https://github.com/bbrowning/snc/commit/6d416a5dca837ef2bf42c6269a2010a239caf965

### CODE REVIEW

File: install-config.yaml [hunk 1]
```diff
@@ -19,13 +19,13 @@ metadata:
...
-  - cidr: 10.128.0.0/14
+  - cidr: 10.116.0.0/14
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.126.0/24
   networkType: OpenShiftSDN
   serviceNetwork:
-  - 172.30.0.0/16
+  - 172.25.0.0/16
 platform:
```
Comment (cfergeau) [inline install-config.yaml:29]: I assume these values were just picked "randomly"?

**Comment by guillaumerose** [comment]
It might worth a release note when released with crc.

/lgtm

File: install-config.yaml (hunk 1 above)
Comment (praveenkumar) [reply install-config.yaml:29]: yes these are randomly picked.

Comment (guillaumerose) [reply install-config.yaml:29]: we could choose a single IP block (large enough) and say explicitly in the documentation that this IP block is reserved for crc. 
Inside this block, we can do what we want.

**Pull request merged**